examples/platform/silabs/SiWx917/uart.cpp

```diff
@@ -118,6 +118,30 @@ int16_t uartConsoleWrite(const char * Buf, uint16_t BufLength)
     return BufLength;
 }
 
+/**
+ * @brief Write Logs to the Uart. Appends a return character
+ *
+ * @param log pointer to the logs
+ * @param length number of bytes to write
+ * @return int16_t Amount of bytes written or ERROR (-1)
+ */
+int16_t uartLogWrite(const char * log, uint16_t length)
+{
+    if (log == NULL || length < 1)
+    {
+        return UART_CONSOLE_ERR;
+    }
+    for (int i = 0; i < length; i++)
```

Comment:
should actually match the type of `length`
```suggestion
    for (uint16_t i = 0; i < length; i++)
```